src/components/TodoList/TodoList.tsx

```diff
@@ -1,100 +1,129 @@
+function filterTodos(status: StatusOfFilter | string, todos: Todo[]): Todo[] {
+  switch (status) {
+    case StatusOfFilter.All:
+      return todos.filter(todo => todo);
 
-      <tr data-cy="todo" className="">
-        <td className="is-vcentered">6</td>
-        <td className="is-vcentered" />
-        <td className="is-vcentered is-expanded">
-          <p className="has-text-danger">
-            qui ullam ratione quibusdam voluptatem quia omnis
-          </p>
-        </td>
-        <td className="has-text-right is-vcentered">
-          <button data-cy="selectButton" className="button" type="button">
-            <span className="icon">
-              <i className="far fa-eye" />
-            </span>
-          </button>
-        </td>
-      </tr>
+    case StatusOfFilter.Active:
+      return todos.filter(todo => !todo.completed);
+
+    case StatusOfFilter.Completed:
+      return todos.filter(todo => todo.completed);
+
+    default:
+      return todos;
+  }
+}
 
-      <tr data-cy="todo" className="">
-        <td className="is-vcentered">8</td>
-        <td className="is-vcentered">
-          <span className="icon" data-cy="iconCompleted">
-            <i className="fas fa-check" />
-          </span>
-        </td>
-        <td className="is-vcentered is-expanded">
-          <p className="has-text-success">quo adipisci enim quam ut ab</p>
-        </td>
-        <td className="has-text-right is-vcentered">
-          <button data-cy="selectButton" className="button" type="button">
+export const TodoList: React.FC<Props> = ({
+  todos,
+  fnSelectTodo,
+  selectTodoId,
+  fnSelectUser,
+  status,
+}) => {
+  const list: Todo[] = useMemo(() => {
+    return filterTodos(status, todos);
+  }, [todos, status]);
+
+  const showTodoModal = (id: number, userId: number) => {
+    fnSelectTodo(id);
+    fnSelectUser(userId);
+  };
+
+  return (
+    <table className="table is-narrow is-fullwidth">
+      <thead>
+        <tr>
+          <th>#</th>
+          <th>
             <span className="icon">
-              <i className="far fa-eye" />
+              <i className="fas fa-check" />
             </span>
-          </button>
-        </td>
-      </tr>
-    </tbody>
-  </table>
-);
+          </th>
+          <th>Title</th>
+          <th> </th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {list.map(({
+          id,
+          completed,
+          title,
+          userId,
+        }) => (
+          <tr
+            data-cy="todo"
+            className={classNames(
+              { 'has-background-info-light': id === selectTodoId },
+            )}
+            key={id}
+          >
+            <td className="is-vcentered">{id}</td>
+            <td className="is-vcentered">
+              { completed && (
+                <span className="icon">
+                  <i className="fas fa-check" />
+                </span>
+              )}
+            </td>
+            <td className="is-vcentered is-expand">
```

Comment:
Replace 'is-expand' with 'is-expanded' for consistency with the Bulma CSS framework.
![image](https://user-images.githubusercontent.com/96409563/230784368-867d13e1-46df-43c9-b9a7-ba0fcee85d32.png)